common/config/config-distribution.rb

```diff
@@ -63,6 +63,9 @@ def self.get_preferred_config_path
     if java.lang.System.getProperty("aspace.config")
       # Explicit Java property
       java.lang.System.getProperty("aspace.config")
+    elsif ENV['ASPACE_CONFIG'] && File.exist?(ENV['ASPACE_CONFIG'])
```

Comment:
@cfitz shouldn't the second half in the && clause be File.exist?(File.join(ENV['ASPACE_CONFIG'], "config.rb"))?  Your comment says you are only setting the path so don't you have to add the filename like it is in the other if statements in this section of the code?  Or do you need to change the comment to include the complete path and filename in ENV['ASPACE_CONFIG']?